images/linux/Ubuntu2004-Readme.md

```diff
@@ -72,11 +68,11 @@
 - Git LFS 3.1.4 (apt source repository: https://packagecloud.io/install/repositories/github/git-lfs)
 - Git-ftp 1.6.0
 - Haveged 1.9.1
-- Heroku 7.60.1
-- HHVM (HipHop VM) 4.158.0
+- Heroku 7.60.2
+- HHVM (HipHop VM) 4.159.0
 - jq 1.6
 - Kind 0.12.0
-- Kubectl 1.23.6
+- Kubectl 4.5.4
```

Comment:
@ddobranic is this version ok? This bump looks a bit weird for me

Reply:
I think we can set up it manually and fix in the next rollout:

```
$ ./kubectl version --client --short
Flag --short has been deprecated, and will be removed in the future. The --short output will become the default.
Client Version: v1.24.0
Kustomize Version: v4.5.4
```

Reply:
created an issue
https://github.com/actions/virtual-environments-internal/issues/3680